pkg/cmd/workspace/code.go

```diff
@@ -86,12 +91,38 @@ var CodeCmd = &cobra.Command{
 			ideId = ideFlag
 		}
 
-		view_util.RenderInfoMessage("Opening the workspace in your preferred IDE")
+		view_util.RenderInfoMessage("Opening the workspace project '" + projectName + "' in your preferred IDE.")
 
 		openIDE(ideId, activeProfile, workspaceId, projectName)
 	},
 }
 
+func SelectWorkspaceProject(workspaceId string, profile *config.Profile) (*string, error) {
```

Comment:
No need for this method to be public. Mark it as private to the package by starting with a lower case letter.